sdks/java/io/pom.xml

```diff
@@ -32,6 +32,17 @@
   <description>Beam SDK Java IO provides different connectivity components
   (sources and sinks) to consume and produce data from systems.</description>
 
+  <properties>
+    <!-- standard binary for kubectl -->
+    <kubectl>kubectl</kubectl>
```

Comment:
I'm thinking we could use this infrastructure outside of the `io` module, and outside Java, no? Like root pom.